### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server  Files Editor_Color Contrast: Color contrast ratio is <4.5:1 for HTML code 

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#171]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)

MAS ID Violated: MAS 1.4.3

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till Microsoft IIS on top navigation bar and press enter
5.tab till web server and press enter
6.Tab till Files and press enter
7.Select any files from the list and press enter till to open html code page.(Ex-inetpub/custerr/en-US/401-1.htm/)
8.open color contrast analyzer  and check the color contrast is>=4.5:1 or not

Actual Result:
Color contrast ratio is <=4.5:1 for HTML code 

Expected Result:
Color contrast should be >=4.5:1

User Impact:
Low vision people will face difficulty if the color contrast is less than 4.5:1

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
![bug_171](https://user-images.githubusercontent.com/42638683/44785742-927b9800-abaf-11e8-94fe-85b409a08d3a.png)

**Comment by jhkimnew** [comment]
I think we are using Monaco editor with the default theme. 
This issue will be fixed by Monaco editor in the future. So, closing this issue...

@jimmyca15: Would you confirm this? If you think we can fix this issue, please reactive this.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ColorContrast; #MAS1.4.3; #C+E-H; #.NETWebsites; #Edge

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
